services/apps/data_sink_worker/src/jobs/processOldResults.ts

```diff
@@ -30,13 +32,13 @@ export const processOldResultsJob = async (
 
   const loadNextBatch = async (): Promise<string[]> => {
     return await processWithLock(redis, 'process-old-results', 5 * 60, 3 * 60, async () => {
-      const resultIds = await repo.getOldResultsToProcess(5)
+      const resultIds = await repo.getOldResultsToProcess(MAX_CONCURRENT_PROMISES)
```

Comment:
I would load like 10 at once and process them MAX_CONCURRENT_PROMISES at a time.